.circleci/config.yml

```diff
@@ -216,6 +229,13 @@ jobs:
               package_cloud push choria/${REPO}/el/${d} ${i}
             done
           done
+          for d in buster
+          do
+            for i in $(find /tmp/workspace -name \*.${d}.\*.deb)
+            do
+              package_cloud push choria/${REPO}/debian/${d} ${i}
```

Comment:
This path might also need adjustment.

Reply:
yeah we'll see - these CI definitions are a pain, ok by me if they fail a night or two while we get it right. 

Anticipating issues maybe stick a `find /tmp/workspace` in there just to help us debug so we know whats there